src/db-fixture-pg.ts

```diff
@@ -31,108 +21,100 @@ export class PGFixture
-                    tool1_id, tool1_name, area_id, member_id )
+            this.addTool(db, area_id, member_id, (tool1_id, tool1_name) =>
+                tool1_callback(tool1_id, tool1_name, area_id, member_id),
             );
         };
-        let area_callback = (area_id) => {
-            this.addMember( db,
-                (member_id) => member_callback( member_id, area_id ) );
+        let area_callback = area_id => {
+            this.addMember(db, member_id =>
+                member_callback(member_id, area_id),
+            );
         };
 
-        this.addArea( db, area_callback );
+        this.addArea(db, area_callback);
     }
 
-
-    private addArea(
-        db
-        ,success_callback: ( area_id: number ) => void
-    ): void
-    {
-        db.query({
-            text: [
-                "INSERT INTO areas ("
-                    ,"area_name"
-                ,") VALUES ($1) RETURNING id"
-            ].join( " " )
-            ,values: [
-                "test_area"
-            ]
-        }, (err, res) => {
-            if( err ) {
-                throw new Error( "Error inserting new area into database: "
-                    + err.toString() );
-            }
-            else {
-                success_callback( res.rows[0].id );
-            }
-        });
+    private addArea(db, success_callback: (area_id: number) => void): void {
+        db.query(
+            {
+                text: [
+                    'INSERT INTO areas (',
+                    'area_name',
+                    ') VALUES ($1) RETURNING id',
+                ].join(' '),
+                values: ['test_area'],
+            },
+            (err, res) => {
+                if (err) {
+                    throw new Error(
+                        'Error inserting new area into database: ' +
+                            err.toString(),
+                    );
+                } else {
+                    success_callback(res.rows[0].id);
+                }
+            },
+        );
     }
 
-    private addMember(
-        db
-        ,success_callback: ( member_id: number ) => void
-    ): void
-    {
-        db.query({
-            text: [
-                "INSERT INTO members ("
-                    ,"full_name"
-                ,") VALUES ($1) RETURNING id"
-            ].join( " " )
-            ,values: [
-                "Test Member"
-            ]
-        }, (err, res) => {
-            if( err ) {
-                throw new Error( "Error inserting new member into database: "
-                    + err.toString() );
-            }
-            else {
-                success_callback( res.rows[0].id );
-            }
-        });
+    private addMember(db, success_callback: (member_id: number) => void): void {
+        db.query(
+            {
+                text: [
+                    'INSERT INTO members (',
+                    'full_name',
+                    ') VALUES ($1) RETURNING id',
```

Comment:
This might be tough for this prettifier to handle. The `'full_name'` parameter here was given an extra layer of indent, because it makes sense for the SQL. The prettifier isn't likely to grok embedded SQL like that, though. I'm guessing there's some way to say "PRETTIFIER LEAVE THIS BLOCK ALONE", but that would be tedious to add to all the embedded SQL in here.

I don't like it, but can live with it.

Reply:
Yeah, you can use inline comments to ignore specific lines (or "nodes"): https://prettier.io/docs/en/ignore.html#javascript